modelon/impact/client/entities/workspace.py

```diff
@@ -22,7 +23,9 @@
 logger = logging.getLogger(__name__)
 
 ExperimentDefinition = Union[
-    SimpleModelicaExperimentDefinition, SimpleFMUExperimentDefinition, Dict[str, Any],
+    SimpleModelicaExperimentDefinition,
+    SimpleFMUExperimentDefinition,
+    Dict[str, Any],
```

Comment:
Similar changes has been made to master. You are probably aware but merging into dev and rebasing this one would be good.

Reply:
Ah yes. Sure!